clients/cobol-lsp-vscode-extension/package.json

```diff
@@ -405,6 +405,19 @@
             "iso646jp"
           ]
         },
+        "cobol-lsp.cpy-manager.endevor-dependencies": {
+          "type": "string",
+          "enum": [
+            "ZOWE",
+            "ENDEVOR_PROCESSOR"
+          ],
+          "enumDescriptions": [
+            "ZOWE paths to retrieve copybooks",
```

Comment:
"Zowe paths" is not terminology we use anywhere in the app or documentation so I wouldn't recommend using it here. 

I would make the two options:

COBOL LS 
ENDEVOR

enumDescriptions:
Retrieve copybooks from locations specified in the COBOL LS extension settings and processor groups.
Retrieve copybooks from locations specified in Endevor processor groups.

Reply:
I agree that the description could be better. Still the two options should stay as

- ZOWE
- ENDEVOR_PROCESSOR

because in the first case ZOWE Explorer is used to retrieve the copybooks and in the second case Explorer for Endevor is used.

Soon we will have a third option ENDEVOR_ACMQ when which will differ from ENDEVOR_PROCESSOR in the sense that the source of configuration will not be a processor, but an ACMQ query. In both cases Explorer for Endevor will be used for the download.

The suggested enumDescriptions:

Use ZOWE Explorer to retrieve copybooks from mainframe data sets and USS files specified in the COBOL Language Support extension settings.
Use Explorer for Endevor to retrieve copybooks from locations specified in the Endevor element processor group.

What do you think @zeibura ?

Reply:
Zowe Explorer doesnt have Zowe in all caps, but otherwise fine